spec/index.html

```diff
@@ -32,12 +32,12 @@
     }],
 
     localBiblio: {
-      "I-D.ietf-httpapi-yaml-mediatypes": {
+      "RFC9512": {
```

Comment:
The local "RFC9512" (and "YAML" below) can likely be removed since they are in specref already:
https://www.specref.org/?q=RFC9512
https://www.specref.org/?q=YAML

Reply:
Removed RFC9512. specref YAML references 1.2 and not 1.2.2 though